defaults.yaml

```diff
@@ -68,3 +68,7 @@ loom:
 # asset server (must NOT be localhost because it's called from within container)
 assets:
   url: "http://localhost:3070"
+
+# auth credentials service (i.e., for stash)
+auth:
```

Comment:
Any OAuth1 service could need that, so keeping the interface generic. That's why this endpoint returns the full value of the `Authorization` header to keep implementation details abstracted.